resources/views/navbar-notifications.blade.php

```diff
@@ -25,7 +28,7 @@
                             </span>
 
                             <div class="flex flex-row space-x-4">
-                                @isset($notification->data['action'])
+                                @isset($notification->data['action']['url'])
```

Comment:
won't this crash if there is no `data['action']` property

Reply:
no